lib/camp.js

```diff
@@ -745,25 +745,45 @@ function Path(matcher) {
 }
 
 // Path unit.
```

Comment:
Could you add a comment to warn that calling `pathLikeUnit()` with its third parameter more than once may reset `server[some method]`? Maybe describe the use of the `httpMethods` parameter.